pcsx2/ImGui/ImGuiOverlays.cpp

```diff
@@ -241,10 +242,14 @@ __ri void ImGuiManager::DrawPerformanceOverlay(float& position_y, float scale, f
 			const float target_speed = VMManager::GetTargetSpeed();
 			const bool is_normal_speed = (target_speed == EmuConfig.EmulationSpeed.NominalScalar);
 			if (!is_normal_speed)
-			{
-				const bool is_slowmo = (target_speed < EmuConfig.EmulationSpeed.NominalScalar);
-				DRAW_LINE(standard_font, is_slowmo ? ICON_FA_FORWARD : ICON_FA_FAST_FORWARD, IM_COL32(255, 255, 255, 255));
-			}
+			  {
+			      if (target_speed == EmuConfig.EmulationSpeed.SlomoScalar) //	Slow-Motion
+			          DRAW_LINE(standard_font, ICON_PF_SLOW_MOTION, IM_COL32(255, 255, 255, 255));
+			      else if (target_speed == EmuConfig.EmulationSpeed.TurboScalar)  // Turbo (Fast-Forward)
+			          DRAW_LINE(standard_font, ICON_FA_FAST_FORWARD, IM_COL32(255, 255, 255, 255));
+			      else // 200% Speed
```

Comment:
comment should be `// Unlimited`

Reply:
Fixed

Reply:
No you haven't?

Reply:
oh also can you fix the indenting please

Reply:
oh shoot

Reply:
Fixed for realsies this time